src/nvim/normal_cmds.lua

```diff
@@ -0,0 +1,1157 @@
+local bit = require('bit')
+
+local NONE = 0
+local NCH = 0x01
+local NCH_NOP = bit.bor(0x02, NCH)
+local NCH_ALW = bit.bor(0x04, NCH)
+local LANG = 0x08
+local SS = 0x10
+local SSS = 0x20
+local STS = 0x40
+local RL = 0x80
+local KEEPREG = 0x100
+local NCW = 0x200
+
+local SEARCH_REV = 0x01
+local BL_WHITE = 1
+local BL_FIX = 4
+
+local FORWARD = 1
+local BACKWARD = -1
+
+local TRUE = 1
+
+local MSCR_DOWN = 0
+local MSCR_UP = 1
+local MSCR_LEFT = -1
+local MSCR_RIGHT = -2
+
+local module = {}
+
+module.nv_cmds = {
+  {
+    cmd_char = 'NUL',
+    cmd_func = 'nv_error',
+    cmd_flags = NONE,
+    cmd_arg = NONE,
+  },
+  {
+    cmd_char = 'Ctrl_A',
```

Comment:
This maps to an internal C macro, right? A major goal of https://github.com/neovim/neovim/issues/19918 is to allow us to expose these Lua data files in the runtime, for use by plugins/users. So exposing a new way to represent keycodes isn't a good idea. Can we use the standard keycode convention here? Or a literal char?

Reply:
yup defined in ascii_defs.h

Reply:
thought ` char = '\x01'  -- ASCII value for Ctrl-A` or `char = '<C-a>' -- Using Neovim's keycode notation`